src/System.Private.ServiceModel/src/project.json

```diff
@@ -50,12 +50,14 @@
   },
```

Comment:
So that outerloop failure is because you're still using an old version of System.Net.WinHttpHandler.  Let me have a look who's bringing that in.